src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -1339,22 +1330,20 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 
 	private _reconnectToTerminals(): void {
 		if (this._hasReconnected) {
-			this._logService.trace(`Already reconnected, to ${this._reconnectedTerminals?.length} terminals so returning`);
+			this._logService.info(`Already reconnected, to ${this._reconnectedTerminals?.length} terminals so returning`);
```

Comment:
We should avoid info as it will show up all the time

Reply:
Yes I'll fix that Monday